include/geode/model/helpers/ray_tracing.h

```diff
@@ -47,4 +47,10 @@ namespace geode
             const BRep& brep,
             const Block3D& block );
 
+    bool opengeode_model_api test_point_inside_block(
+        const BRep& brep, const Block3D& block, const Point3D& point );
+
+    absl::optional< uuid > opengeode_model_api block_containing_point(
```

Comment:
What happens if the point is on a surface or on a line? How many uuid should we return?

Reply:
if the point is on the boundary it is not in the block. 
Here the function return the block containing the point. 
Should we do a more generic function?